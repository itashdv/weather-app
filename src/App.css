.App {
  text-align: center;
}

.App-header {
  text-align: center;
}

.search-box {
  width: 300px;
  margin-left: auto;
  margin-right: auto;
}

.search-input {
  width: 100%;
}

.error {
  color: red;
}

.button {
  cursor: pointer;
  margin-left: 16px;
}

.linkButton {
  background: none!important;
  border: none;
  padding: 0!important;
  margin: 0!important;
  color: #069;
  text-decoration: underline;
  cursor: pointer;
}
