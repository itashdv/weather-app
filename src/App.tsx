import React from 'react';
import './App.css';

import CurrentLocation from './components/current-location';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <p>Welcome to Aitash Weather App!</p>
        <CurrentLocation />
      </header>
    </div>
  );
}

export default App;
