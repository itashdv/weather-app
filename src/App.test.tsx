import { render } from '@testing-library/react';

import App from './App';

it('renders app component', () => {
  const { asFragment } = render(<App />);

  expect(asFragment()).toMatchSnapshot();
});
