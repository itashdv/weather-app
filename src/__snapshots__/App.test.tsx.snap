// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app component 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ fjLCoJ"
  >
    <header
      class="sc-idiyUo oDHss"
    >
      <h4
        class="sc-iBkjds dXoUni"
        role="heading"
      >
        Welcome to Weather App!
      </h4>
    </header>
    <div
      class="sc-gsnTZi JOiGb"
    >
      <input
        aria-controls="locations-listbox"
        aria-expanded="false"
        aria-label="Searchbox for searching locations"
        class="sc-dkzDqf kQOVSO"
        placeholder="Enter the name of the location here.."
        role="combobox"
        type="search"
        value=""
      />
      <div
        class="sc-hKMtZM dtmaCZ"
      />
    </div>
    <div
      class="sc-bjUoiL fBGpkI"
    >
      <p
        class="sc-gKXOVf gDJYYG"
        role="alert"
      >
        Geolocation is not supported by your browser!
      </p>
    </div>
  </div>
</DocumentFragment>
`;
